base_custom_info/models/custom_info.py

```diff
@@ -119,7 +119,7 @@ class CustomInfoValue(models.Model):
     _description = "Custom information value"
     _name = "custom.info.value"
     _rec_name = 'value'
-    _order = "property_id"
+    _order = "name"
```

Comment:
I think this should be ordered by a sequence defined in the template (not sortable at user level).